client/dashboard/me/security-account-recovery/recovery-email.tsx

```diff
@@ -0,0 +1,224 @@
+import {
+	profileQuery,
+	accountRecoveryQuery,
+	updateAccountRecoveryEmailMutation,
+	removeAccountRecoveryEmailMutation,
+	resendAccountRecoveryEmailValidationMutation,
+} from '@automattic/api-queries';
+import { useMutation, useQuery } from '@tanstack/react-query';
+import {
+	__experimentalHStack as HStack,
+	__experimentalInputControl as InputControl,
+	__experimentalVStack as VStack,
+	__experimentalSpacer as Spacer,
+	__experimentalConfirmDialog as ConfirmDialog,
+	Button,
+	Card,
+	CardBody,
+} from '@wordpress/components';
+import { useDispatch } from '@wordpress/data';
+import { DataForm } from '@wordpress/dataviews';
+import { __ } from '@wordpress/i18n';
+import { store as noticesStore } from '@wordpress/notices';
+import { useMemo, useState, useEffect } from 'react';
+import Notice from '../../components/notice';
+import { SectionHeader } from '../../components/section-header';
+import type { Field } from '@wordpress/dataviews';
+
+type SecurityEmailFormData = {
+	email: string;
+};
+
+export default function RecoveryEmail() {
+	const [ formData, setFormData ] = useState< SecurityEmailFormData >( {
+		email: '',
+	} );
+	const [ isRemoveDialogOpen, setIsRemoveDialogOpen ] = useState( false );
+	const [ showResendButton, setShowResendButton ] = useState( true );
+
+	const { data: accountRecoveryData, isLoading: isAccountRecoveryDataLoading } = useQuery(
+		accountRecoveryQuery()
+	);
+
+	const { mutate: validateEmailMutation, isPending: isValidateEmailPending } = useMutation(
```

Comment:
Small nit: the entire object is a "mutation"; so the `mutate` prop should be the action/verb, e.g.


```suggestion
	const { mutate: validateEmail, isPending: isValidateEmailPending } = useMutation(
```

(this is just a convention that I notice in the rest of the codebase.)

Reply:
Makes sense. Updated it here: https://github.com/Automattic/wp-calypso/pull/105504/commits/1255f0357fb13037ded38e60c4c4b52473d40e97